libRALFit/src/ral_nlls_workspaces.f90

```diff
@@ -41,7 +65,13 @@ module ral_nlls_workspaces
 !    summary for every iteration, = 2 gives a summary of the inner iteration
 !    for each iteration, >= 3 gives increasingly verbose (debugging) output
 
-     INTEGER :: print_level = 0
+     INTEGER :: print_level = 3
```

Comment:
The default should be no printing (most people will use this as a black-box deep in code, and won't really care what the algorithm is doing)

Reply:
Changed: print level is by default zero